src/common/Uno.Common/Diagnostics/PlatformDetection.cs

```diff
@@ -17,6 +17,65 @@ public static class PlatformDetection
         public static readonly bool IsArm;
         public static readonly bool Is64Bit;
 
+        public static string HomeDirectory
+        {
+            get
+            {
+                // Ported from https://github.com/sindresorhus/os-homedir/blob/master/index.js
+
+                var home = Environment.GetEnvironmentVariable("HOME");
+
+                if (IsWindows)
+                {
+                    var userProfile = Environment.GetEnvironmentVariable("USERPROFILE");
+                    if (!string.IsNullOrEmpty(userProfile))
+                        return userProfile;
+
+                    var homePath = Environment.GetEnvironmentVariable("HOMEDRIVE") +
+                                   Environment.GetEnvironmentVariable("HOMEPATH");
+                    if (!string.IsNullOrEmpty(homePath))
+                        return homePath;
+
+                    if (!string.IsNullOrEmpty(home))
+                        return home;
```

Comment:
I think it's better to always respect $HOME when it's set. It indicates that the user explicitly wants that folder to be treated as the home directory. In fact, not doing so would be a regression, I think.

Reply:
Node.js believes otherwise and always respect $USERPROFILE on Windows (https://nodejs.org/api/os.html#os_os_homedir).

But you're probably right. We always respected $HOME before and we don't want regressions.